packages/preview/bone-resume/0.2.0/lib.typ

```diff
@@ -0,0 +1,55 @@
+#let resume-init(title: none, author: "六个骨头", header: none, footer: none, body) = {
+  set document(author: author, title: title)
+  set page(margin: (x: 3.2em, y: 3.2em), header: header, footer: footer)
+  set text(font: ("Source Han Sans"), lang: "zh")
+  show emph: it => {
+    text(it.body, font: ("Source Han Serif SC",), style: "italic")
+  }
+  show link: it => {
+    set text(blue, font: ("Source Han Sans"), weight: "bold")
+    it
+  }
+  show heading.where(level: 1): it =>{
+    set text(rgb("#448"), size: 1em, font: "Source Han Sans")
+    stack(dir: ttb, spacing: 0.55em, {
+      it.body
+    }, line(stroke: 1.5pt, length: 100%))
+    v(0.5em, weak: true)
+  }
+  show par: set block(spacing: 0.65em)
+
+  align(center)[
+    #block(text(weight: 700, 1.75em, title))
+  ]
+  set par(justify: true)
+  body
+}
+
+#let primary_achievement(name, decs, contrib) = {
```

Comment:
To be consistent with other functions, you may want to rename that one to `primary-achievement`.

Reply:
Okay, it has been modified, thank you.